examples/va/nvidia-mdev/edpm/deployment/kustomization.yaml

```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+components:
+  - ../../../../../../va/nvidia-mdev/edpm/deployment
```

Comment:
Relative parent needs to be removed: `../../../../../va/nvidia-mdev/edpm/deployment`